apps/web-giddh/src/app/theme/sales-ng-virtual-select/sh-select.component.scss

```diff
@@ -171,7 +171,7 @@ button.form-control:focus {
 
 .bottom-border-only {
     border: none !important;
-    border-bottom: 1px solid $light-gray !important;
```

Comment:
@aditi-tiwari17 @shalinee-walkover Please check this code revert

Reply:
checked, its fine